lisa/tools/xfstests.py

```diff
@@ -1,17 +1,30 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
 import re
+from dataclasses import dataclass
 from pathlib import Path, PurePath
-from typing import Any, List, Type, cast
+from typing import TYPE_CHECKING, Any, Dict, List, Type, cast
 
+from lisa import notifier
 from lisa.executable import Tool
+from lisa.messages import CommunityTestMessage, TestStatus, create_test_result_message
 from lisa.operating_system import CBLMariner, Debian, Posix, Redhat, Suse, Ubuntu
 from lisa.tools import Cat, Echo
 from lisa.util import LisaException, UnsupportedDistroException, find_patterns_in_lines
 
 from .git import Git
 from .make import Make
 
+if TYPE_CHECKING:
```

Comment:
Move this tool to testsuites/xfstests, because it causes some cross reference, and it's actually not be used by multiple test suites.

Reply:
moved